apps/gro/fields/address-outside.js

```diff
@@ -1,19 +1,8 @@
 'use strict';
 
 module.exports = {
-  'address-text-one': {
+  'address-text': {
     validate: ['required'],
-  },
-  'address-text-two': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-three': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-four': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-five': {
-    labelClassName: 'visuallyhidden'
+    'ignore-defaults': true
```

Comment:
This stops the default formatters being applied - https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/form.js#L19

The param is in formatters/index - https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/formatting/index.js#L20

This formatter is applied by default and gets rid of line breaks etc - https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/formatting/index.js#L20

Another hidden feature